pkg/remote/remote.go

```diff
@@ -113,15 +123,23 @@ func Referrers(ctx context.Context, d name.Digest, option types.RegistryOptions)
 	return nil, errs
 }
 
-func httpTransport(insecure bool) *http.Transport {
+func httpTransport(insecure bool, clientCert []byte, clientKey []byte) (*http.Transport, error) {
```

Comment:
There are three arguments now. We may want to add RegistryOptions instead.

```suggestion
func httpTransport(option types.RegistryOptions) (*http.Transport, error) {
```